webpack/webpack.config.server.js

```diff
@@ -52,5 +52,16 @@ module.exports = webpackConfig({
         {},
       ),
     }),
+    ...(process.env.SENTRY_AUTH_TOKEN
+      ? [
+          new SentryWebpackPlugin({
+            release: process.env.HEROKU_SLUG_COMMIT,
```

Comment:
Is this env variable `HEROKU_SLUG_COMMIT` needed to test the integration locally? Do we need to able to test it locally after we've merged? And if so, should `HEROKU_SLUG_COMMIT` be added to `.env.example`? 🤔 

And maybe the same questions goes for the `SENTRY_AUTH_TOKEN` env variable?

Reply:
Ahh yeah I could add those there and comment that they are optional :-)

Reply:
Awesome!! 🙌 😀